src/sagemaker/fw_utils.py

```diff
@@ -59,7 +59,7 @@
     "local_gpu",
 )
 SM_DATAPARALLEL_SUPPORTED_FRAMEWORK_VERSIONS = {
-    "tensorflow": ["2.3.0", "2.3.1", "2.4.1"],
+    "tensorflow": ["2.3.1", "2.3.2", "2.4.1"],
```

Comment:
Is this removing 2.3.0 as a valid version?

Reply:
Correct. TF 2.3.0 images are not usable with SM DataParallel